tests/functional/test_rose_stem.py

```diff
@@ -244,7 +244,6 @@ def _inner_fn(rose_stem_cmd, verbosity=verbosity):
             **setup_stem_repo
         }
     yield _inner_fn
-    shutil.rmtree(setup_stem_repo['suite_install_dir'])
```

Comment:
I'm not sure this should be removed. It's a workflow run dir, right (not in `temp_path`)? So doesn't get cleaned up any other way unless I'm missing something?

Reply:
stet 
```suggestion
     shutil.rmtree(setup_stem_repo['suite_install_dir']) 
```